.github/ISSUE_TEMPLATE/handbooks/Project_Advisory_Guideline.md

```diff
@@ -0,0 +1,43 @@
+
+- You understand the roles and responsibilities of the PA 
+- You have enough time to dedicate as it might be needed. 
+- Get familiar yourself with the Handbook, Guidelines and general flow of the Brainhack.
+- Create your Mattermost account and enrol to the [Project Advisors Channel]() to convey further discussions. 
+- Share some details about yourself in the [PA Spreadsheet]().
+
+## Roles and Responsibilities
+
+Your roles from the stage of project submission to the end day of the Brainhack includes;
+
+- Reviewing project assigned with you or you have chosen
+- Conveying a continuous communication with the project leaders and members through the Brainhack to;
+- If you would like, share your thoughts/advice on the methodological, technological and theoretical aspects of the projects.
+  - Help them to find and contact with the experts from the community in the case of queries.
+  - Direct them to related Brainhack sources/tools to use/read.
+  - Check that there is no conflict or violation of CoC occurs among the project members. 
+-  Run daily and brief project meetings to observe how the project is going, share your advice and suggestions, and check if there is any need of help regarding the project.
+
+### At the Beginning of the Project Submission
+
+1. Review the issue associated with the project and check whether the details listed in the checklist were provided or not.
+2. Contact with the project leader through the PR and declare that you are PA for this project and your role in the project through the 3 days of Brainhack as it is listed below. 
+3. If there is any missing information in the project submission issue, ask the project leader to provide the information. 
+4. Fill the PA sheet regarding the project/s you will be PA for.
+5. Fill the PA sheet regarding whether first initial contact with the project leader was made and you explained your role to them.
+6. Review the project start, teaming up and share your decision in the related area of the PA sheet. 
+
+
+### After the Project is Accepted
+1. Make the acceptance declaration to the project leader.
+2. Make sure they have prepared a pitching presentation comply with the requirements of the Brianhack guidelines.
+3. Make daily meetings with the project team members to discuss the progresses in the project and understand if there are any obstacles along to the way that requires OHBM Team’s or experts involvements. 
+4. Issue the progress of the projects in the spreadsheet. 
+
+
+### After the Brainhack Finishes
+
+1. Guide them towards the demo pitch that will be held at the end of the Brainhack.
+2. Mention to the project members the proceeding possibilities for the Brainhack projects that might lead them to a publication in the future. 
```

Comment:
I've seen some discussions about this on mattermost, and AFAIK, there is no clear venue for the proceedings yet, so unless this is clear before the hackathon, I would remove it.